nodeJS/authentication/authentication_basics.md

```diff
@@ -1,62 +1,62 @@
 ### Introduction
 
-Creating users and allowing them to log in and out of your web apps is a crucial functionality that we are finally ready to learn! There is quite a bit of setup involved here, but thankfully none of it is too tricky. You'll be up and running in no time! In this lesson, we're going to be using [passportJS](https://www.passportjs.org), an excellent middleware to handle our authentication and sessions for us.
+Creating users and allowing them to log in and out of your web apps is a crucial functionality that we are finally ready to learn! There is quite a bit of setup involved here, but thankfully none of it is too tricky. You'll be up and running in no time! In this lesson, we're going to be using [passport.js](https://www.passportjs.org), an excellent middleware to handle our authentication and sessions for us.
 
 We're going to be building a very minimal express app that will allow users to sign up, log in, and log out. For now, we're just going to keep everything except the views in one file to make for easier demonstration, but in a real-world project, it is best practice to split our concerns and functionality into separate modules.
 
 ### Lesson overview
 
 This section contains a general overview of topics that you will learn in this lesson.
 
-- Understand the use order for the required middleware for PassportJS.
-- Describe what PassportJS Strategies are.
+- Understand the use order for the required middleware for Passport.js.
+- Describe what Passport.js Strategies are.
 - Use the LocalStrategy to authenticate users.
 - Explain the purpose of cookies in authentication.
 - Review prior learning material (routes, templates, middleware, async/await, and promises).
-- Use PassportJS to set up user authentication with Express.
+- Use Passport.js to set up user authentication with Express.
 - Describe what bcrypt is and its use.
 - Describe what a hash is and explain the importance of password hashing.
 - Describe bcrypt's `compare` function.
 
 ### Set up
 
-We're going to be using another Mongo database, so before we begin log in to your mongo provider and create a new database and save its URL string somewhere handy.
+Before we start, create a new database or use your existing one that you've created in the Using PostgreSQL lesson.
```

Comment:
I'd rather have them create a new database. More practice for the learners.

Reply:
Done